### PR TITLE
[hotfix][runtime]fix typo in HistoryServerUtils class

### PR DESCRIPTION
**Pull request by zhisheng17**
## What is the purpose of the change

fix typo in HistoryServerUtils class

### CODE REVIEW

**Review by xiaoHoly — COMMENTED** [review]
LGTM

**Comment by zhisheng17** [comment]
the code will error if remove the `;`

**Comment by zhisheng17** [comment]
what about use class instead of enum? @tillrohrmann

**Comment by xiaoHoly** [comment]
hi,@zhisheng17 .I have learned some information, and I think this idea makes sense. Share with you now。https://blog.csdn.net/weixin_39525007/article/details/110726349

File: flink-runtime/src/main/java/org/apache/flink/runtime/webmonitor/history/HistoryServerUtils.java [hunk 1]
```diff
@@ -35,7 +35,6 @@
  * Utility class for the HistoryServer.
  */
 public enum HistoryServerUtils {
```
Comment (V1ncentzzZ) [inline flink-runtime/src/main/java/org/apache/flink/runtime/webmonitor/history/HistoryServerUtils.java:37]: We should change the definition to class if we remove the semicolon.

**Comment by zhisheng17** [comment]
@flinkbot run azure

**Comment by tillrohrmann** [comment]
I think the enum pattern is used here in order to prevent that someone instantiates the `HistoryServerUtils` class. Hence, I am sceptical about the benefit of this change here. If we want to change it, then we should at least make the constructor private or let it throw an exception which prevents instantiating this class. But I am wondering whether spending time on this issue is worth it. I would be in favour of closing this PR if you are ok with it @zhisheng17.

**Comment by zhisheng17** [comment]
thanks @tillrohrmann , I get it, I will close this PR.

**Pull request closed without merging**